baybe/acquisition/base.py

```diff
@@ -61,25 +61,24 @@ def to_botorch(
         import torch
         from botorch.acquisition.objective import LinearMCObjective
 
-        # Get computational data representations
-        train_x = searchspace.transform(measurements, allow_extra=True)
-        train_y = objective.transform(measurements)
-
-        # Retrieve corresponding botorch class
+        # Retrieve botorch acquisition function class and match attributes
         acqf_cls = getattr(bo_acqf, self.__class__.__name__)
-
-        # Match relevant attributes
         params_dict = match_attributes(
             self, acqf_cls.__init__, ignore=self._non_botorch_attrs
         )[0]
 
+        # Create botorch surrogate model
+        bo_surrogate = surrogate.to_botorch()
+
+        # Get computational data representation
+        train_x = to_tensor(searchspace.transform(measurements, allow_extra=True))
+
         # Collect remaining (context-specific) parameters
         signature_params = signature(acqf_cls).parameters
         additional_params = {}
-        if "model" in signature_params:
```

Comment:
Why is this check no longer necessary?

Reply:
@Scienfitz put that in, but the acquisition function *always* has a model (without, the whole endeavor would be a pointless, no? 😄 ). It's in fact part of the signature of the absolut basic `AcquisitionFunction` base class in botorch.

Reply:
yes I verified in the botorch code as well, this change makes sense